packages/svelteui-core/src/components/Table/Table.stories.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+	import { Table } from './index';
+</script>
+
+<Meta title="Components/Table" component={Table} />
+
+<Template let:args>
+	<Table {...args} />
```

Comment:
The table story needs content in it's slot, can you add it? It is useful to see and test